web/frontend/angular/src/app/auth/auth.service.ts

```diff
@@ -15,6 +17,10 @@ export class AuthService {
         localStorage.removeItem("ACCESS_TOKEN")
     }
     public register(userData: User) {
-        //TODO
+        console.log(userData)
```

Comment:
Remove this log. If needed, this data can be seen in the developer console by inspecting the network request in the network tab.